gulp/config.js

```diff
@@ -10,8 +10,7 @@ module.exports = {
         src:  src,
         dev:  dev,
         dist: dist,
-		// set to specific folder with name leading slash, e.g. '/resources-content'
-		additionalResources: false
+		resources: ['/resources']
```

Comment:
Hey @alxbenz danke für den umbau.
Können wir das bitte so umbauen, dass man es nicht im gulp/* Ordner Konfigurieren muss?
Mir schwebt sowas vor wie eine projectConfig.json im Projekt root folder.
Ist diese nicht da, werden default Werte verwendet also in diesem Fall ['/resources']
ist die Datei aber vorhanden und der Wert dort gesetzt überschreibt dieser den default wert